resources/telemetry/charts/fluent-bit/README.md

```diff
@@ -0,0 +1,23 @@
+# Fluent Bit Helm Chart
+
+[Fluent Bit](https://fluentbit.io) is a fast and lightweight log processor and forwarder or Linux, OSX and BSD family operating systems.
+
+## Installation
+
```

Comment:
are these instructions supposed to be done in order, or are they alternatives to each other?